src/lib/Tab/LocationView/PagerLocationToDataMapper.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Tab\LocationView;
+
+use EzSystems\EzPlatformAdminUi\UI\Value\ValueFactory;
+use EzSystems\EzPlatformAdminUi\UI\Value;
+use Pagerfanta\Pagerfanta;
+
+final class PagerLocationToDataMapper
```

Comment:
I'd design this class differently. Instead of making explicit "PagerLocation" based Mapper make it general purpose one. It should accept an array of `Location` objects and return enriched UI Location objects.